gemini/use-cases/code/code_scanning_and_vulnerability_detection_with_geminipro_1.5.ipynb

```diff
@@ -0,0 +1,999 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "nUWIDIzvODfw",
+   "metadata": {
+    "id": "nUWIDIzvODfw"
+   },
+   "source": [
+    "# Code Vulnerability Scanning & Automated Remediation using Vertex AI Gemini API (Gemini Pro 1.5)\n",
+    "\n",
+    "---\n",
```

Comment:
Add "Open in" links to top of notebook. Similar to this structure https://github.com/GoogleCloudPlatform/generative-ai/blob/main/gemini/getting-started/intro_gemini_1_5_flash.ipynb

Reply:
done.

Reply:
done